frontend/src/components/Tabs/index.tsx

```diff
@@ -44,10 +47,11 @@ export default function Tabs({
               content: (
                 <Tab
                   label={tab?.label ?? ' '}
-                  isSelected={index === 1}
+                  isSelected={!!isSelected}
                   onClick={tab?.onClick}
                   icon={tab?.icon || <Square />}
                   alternativeLabels={tab?.alternativeLabels}
+                  iconAlignment={iconAlignment ?? TabIconAlignment.Left}
```

Comment:
А так не можно? У нас же в `Tab` `TabIconAlignment.Left` - default value
```suggestion
                  iconAlignment={iconAlignment}
```